src/main/kotlin/no/brreg/informasjonsforvaltning/organizationcatalogue/controller/OrganizationsApiImpl.kt

```diff
@@ -111,6 +110,14 @@ open class OrganizationsApiImpl(
         }
     }
 
+    override fun updateFromEnhetsregisteret(httpServletRequest: HttpServletRequest?, jwt: Jwt, id: String): ResponseEntity<Organization> =
+        if (endpointPermissions.hasAdminPermission(jwt)) {
+            LOGGER.info("update organization with id $id with data from Enhetsregisteret")
```

Comment:
Det går an å legge på en ekstra sjekk at org nummer er 9 siffer og kaste BadRequest hvis ikke, men er vel ikke så kritisk

Reply:
De ender opp med NotFound når de sender inn noe som ikke er et orgnr, greit nok for vår del. Hindrer potensielt spam til brreg, men det blir rimelig minimalt, iom at det bare er root-brukere som kan bruke det.